query/src/main/scala/filodb/query/exec/SelectRawPartitionsExec.scala

```diff
@@ -18,8 +19,8 @@ object SelectRawPartitionsExec {
   def histMaxColumn(dataset: Dataset, colIDs: Seq[Types.ColumnId]): Option[Int] = {
     colIDs.find { id => dataset.dataColumns(id).columnType == HistogramColumn }
           .flatMap { histColID =>
-            dataset.dataColumns.find { c => c.name == "max" && c.columnType == DoubleColumn }
-          }.map(_.id)
+            dataset.dataColumns.find(_.name == "max").map(_.id)
```

Comment:
Why is the criteria that max be a DoubleColumn removed?  That should remain there.

Reply:
Sure. This was unintentionally removed due to refactor.